Add some crosscompiled binary outputs to the autobuild

hamishcoleman opened the pull request:
Hopefully this will also demonstrate how to do a cross compile with the makefiles.  

The cross compile process is summarised as:
```
HOST_TRIPLET=arm-linux-gnueabi
sudo apt-get install binutils-$HOST_TRIPLET gcc-$HOST_TRIPLET
./autogen.sh
export CC=$HOST_TRIPLET-gcc
export AR=$HOST_TRIPLET-ar
./configure --host $HOST_TRIPLET
make
```

hamishcoleman committed:
Add an example cross compile build

hamishcoleman committed:
Harmonise the naming to reflect full architecture and if it is a real package or not

hamishcoleman committed:
Add some more example cross compile targets

hamishcoleman committed:
Only one RPM package is created, so use the singular word

hamishcoleman committed:
Dont use build triplet for OS packages, use the OS prefered arch name

Logan007 commented:
This looks extremely good to me.

Do you want to a small section to `doc/Building.md` about cross-compilation? Basically your comment from above with one example system / run?

hamishcoleman committed:
Add some cross compiling documentation to the Building.md

hamishcoleman commented:
Documentation is always a good idea.  I've added that.

What did you mean by "example system / run"?  I think I have covered that with the arm-linux-gnueabi example I added, but wanted to be sure I understood you correctly.

Logan007 commented:
Thank you very much!!
